includes/admin/class-amp-template-customizer.php

```diff
@@ -108,8 +108,8 @@ public static function init( WP_Customize_Manager $wp_customize ) {
 
 				add_action( 'customize_controls_print_footer_scripts', [ $self, 'print_legacy_controls_templates' ] );
 				add_action( 'customize_preview_init', [ $self, 'init_legacy_preview' ] );
-
 				add_action( 'customize_controls_enqueue_scripts', [ $self, 'add_legacy_customizer_scripts' ] );
+				add_filter( 'customize_previewable_devices', [ $reader_theme_loader, 'customize_previewable_devices' ] );
```

Comment:
This isn't quite right, as it's causing Tablet to be the initially-previewed device when accessing the Customizer regardless of whether you're looking to open the AMP panel. I'll amend.